apiserver/errors/errors.go

```diff
@@ -74,6 +74,15 @@ var singletonErrorCodes = map[errors.ConstError]string{
 	ErrActionNotAvailable:                        params.CodeActionNotAvailable,
 }
 
+// ParamsError is responsible for constructing a [params.Error] with the given
+// code and formatted error message.
+func ParamsError(code string, format string, a ...any) *params.Error {
+	return &params.Error{
+		Code:    code,
+		Message: fmt.Sprintf(format, a...),
```

Comment:
At first I thought there was no way to set the Info parameter. Then in another file I saw `WithInfo()`.  I wonder if we want these 2 methods next to each other. In other cases we do that - the base constructor and various With helpers are declared together and it's much easier to grok.

Reply:
I am lost on this one. Can we chat more?

Reply:
As discussed, was just wanting to keep the constructor and the WithInfo() methods together.

Reply:
The problem is they are across two seperate packages with different concerns. We have agreed I believe to leave it where it is.